region/brinstar/green/Green Brinstar Fireflea Room.json

```diff
@@ -146,6 +146,28 @@
         }
       }
     },
+    {
+      "link": [2, 2],
+      "name": "Gain Flash Suit (Spikesuit)",
+      "requires": [
+        {"or": [
+          {"canShineCharge": {
+            "usedTiles": 16,
+            "openEnd": 0
+          }},
+          {"and": [
+            {"doorUnlockedAtNode": 2},
+            {"canShineCharge": {
+              "usedTiles": 17,
+              "openEnd": 0
+            }}
+          ]}
+        ]},
+        "canSpikeSuit",
+        {"shinespark": {"frames": 5, "excessFrames": 5}}
+      ],
```

Comment:
needs a thorn hit

Reply:
Done. 

I think in general we might want some kind of leniency helper(s) for spikesuit attempts. But in this spot it seems probably unnecessary since there's a convenient farm in the room and it's only thorns.